src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/caseupdater/CcdBundleCloningService.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.em.orchestrator.service.caseupdater;
+
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.em.orchestrator.service.ccdcallbackhandler.CcdCallbackDto;
+import uk.gov.hmcts.reform.em.orchestrator.service.dto.CcdBundleDTO;
+import uk.gov.hmcts.reform.em.orchestrator.service.dto.CcdValue;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@Transactional
+public class CcdBundleCloningService implements CcdCaseUpdater {
+
+    private final ObjectMapper objectMapper;
+    private final JavaType type;
+
+    public CcdBundleCloningService(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+        type = objectMapper.getTypeFactory().constructParametricType(CcdValue.class, CcdBundleDTO.class);
+    }
+
+    @Override
+    public boolean handles(CcdCallbackDto ccdCallbackDto) {
+        return false;
+    }
+
+    @Override
+    public JsonNode updateCase(CcdCallbackDto ccdCallbackDto) {
+
+        Optional<ArrayNode> maybeBundles = ccdCallbackDto.findCaseProperty(ArrayNode.class);
+        List<JsonNode> updatedBundlesList = new ArrayList<>();
+
+        if (maybeBundles.isPresent()) {
+            for (int i = 0; i < maybeBundles.get().size(); i++) {
+                try {
+                    JsonNode originalJson = maybeBundles.get().get(i);
+
+                    CcdBundleDTO originalBundle = bundleJsonToBundleDto(originalJson);
+                    boolean isEligibleForCloning = originalBundle.getEligibleForCloningAsBoolean();
```

Comment:
personally I wouldn't use this if statement, I would just add a line after the if statement `originalBundle.setEligibleForCloningAsBoolean(false);` - if it was already false then so what.

It would mean the cloned bundle would need to be set to false as well.